index.html

```diff
@@ -5,194 +5,263 @@
                 </li>
-                <li class="tech html">
+
+                <li class="html">
                     <img src="./assets/html.png" alt="">
                     <span>HTML</span>
                 </li>
-                <li class="tech css">
-                    <img src="./assets/css.png" alt="">
-                    <span>CSS</span>
+
+                <li class="js">
+                    <img src="./assets/js.png" alt="">
+                    JavaScript
                 </li>
-                <li class="framework">
-                    <span>Frameworks</span>
-                    <i class='fas fa-angle-right'></i>
+            </ul>
+            <ul class="frameworks">
+                <li class="frameworks-item"><h3>Frameworks</h3>
+                    <i class="fa fa-chevron-right"></i>
                 </li>
-                <li class="skill">
-                    <span>Skills</span>
-                    <i class='fas fa-angle-right'></i>
+            </ul>
+            <ul class="skills">
+                <li class="skills-item"><div><h3>Skills</h3></div>
+                    <div><i class="fa fa-chevron-right"></i></div>
                 </li>
             </ul>
         </div>
     </section>
-    <section class="contact-form">
-        <div class="contact-form-section-wrapper">
-            <div class="text-content">
-                <h2>Contact Me</h2>
-                <p>f you have an application you are interested in
-                developing, a feature that you need built or a project that needs coding. I'd love to help with it!</p>
+    <footer>
+        <section class="footer-section pad-aside">
+            <div class="footer-text">
+                <h1>Contact me</h1>
+                <p>
+                    If you have an application you are interested in developing, a feature that you need built or a project that needs coding.
+                    I'd love to help with it!
+                </p>
             </div>
-            
-            <form action="https://formspree.io/f/mrgvjqgg" method="POST">
-                <input name="name" type="text" placeholder="Full name" maxlength="30" required>
-                <input name="email" type="email" placeholder="Email address" required>
-                <textarea name="message" id="" cols="30" rows="10" placeholder="Write your message here..." maxlength="500" required></textarea>
-                <input type="submit" value="Get in touch">
+            <form action="https://formspree.io/f/xeqwrwvv" method="POST">
+                <input type="text" name="name" maxlength="30" placeholder="Lorem ipsum" >
+                <input type="email" name="email" placeholder="Loremipsum@gmail.com" >
+                <textarea name="message" maxlength="500" id="" cols="30" rows="10" placeholder="Write your message here" required></textarea>
+                <button class="footer-submit-btn" type="submit">Get in touch</button>
             </form>
```

Comment:
- [ ] Kindly give the inputs elements, the texarea inclusive a bit of a left padding on the inside to match the design perfectly.

- Your design.
<img width="596" alt="image" src="https://user-images.githubusercontent.com/41163571/236343841-d9249568-ac62-481f-985c-9a7eea53fadb.png">

- Figma design
<img width="504" alt="image" src="https://user-images.githubusercontent.com/41163571/236343982-474a1b23-b4c4-4b57-8918-324bef938c11.png">

Reply:
That escaped my sight. It should have 16 px left.
Thank you😎